### PR TITLE
[NL|EN|FR|DE|IT|PT|ES Number] Fixed large number expressed as a fraction not recognized correctly (#2786)

### PR DESCRIPTION
**Pull request by aitelint**
Fix to issue #2786 in EN|ES|IT.

Test cases added to NumberModels.

### CODE REVIEW

**Comment by nawanas** [comment]
this should generalize to more than just the half and quarter, the fractions up to a tenth would make sense, including multiples of these like two thirds, and in the other languages supported by number extractors

**Comment by tellarin** [comment]
@aitelint Will you add the changes suggested by @nawanas soon? I'm fine with merging the current and having a separate issue opened for the suggested generalization.

File: .NET/Microsoft.Recognizers.Text.Number/Spanish/Parsers/SpanishNumberParserConfiguration.cs [hunk 1]
```diff
@@ -95,6 +97,20 @@ public override IEnumerable<string> NormalizeTokenSet(IEnumerable<string> tokens
...
             }
 
+            // The following piece of code is needed to compute the fraction pattern number+'y medio'
+            // e.g. 'cinco y medio' ('five and a half') where the numerator is omitted in Spanish.
+            // It works by inserting the numerator 'un' ('a') in the list fracWords
+            // so that the pattern is correctly processed.
+            var resLen = result.Count;
+            if (resLen > 2)
+            {
+                if (result[resLen - 1] == "medio" && result[resLen - 2] == "y")
```
Comment (tellarin) [inline .NET/Microsoft.Recognizers.Text.Number/Spanish/Parsers/SpanishNumberParserConfiguration.cs:107]: Please move any hard coded terms to YAML resources.

**Review by tellarin — CHANGES_REQUESTED** [review]
Should the relevant changes also be added to PT?

**Comment by aitelint** [comment]
> @aitelint Will you add the changes suggested by @nawanas soon? I'm fine with merging the current and having a separate issue opened for the suggested generalization.

I am working on it right now, I think I will be able to push later today, or tomorrow

**Comment by aitelint** [comment]
@tellarin, changes added and generalized to other European languages

File: .NET/Microsoft.Recognizers.Text.Number/Spanish/Parsers/SpanishNumberParserConfiguration.cs (hunk 1 above)
Comment (aitelint) [reply .NET/Microsoft.Recognizers.Text.Number/Spanish/Parsers/SpanishNumberParserConfiguration.cs]: moved

**Review by tellarin — APPROVED** [review]

**Pull request merged**